.npmrc

```diff
@@ -1,3 +0,0 @@
-username=greg-a-smith
-email=greg@joeboo.net
-//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
I believe this file and its value are necessary for publishing.  Can we revert this and then I'll merge the rest?

Reply:
Sure - I couldn't `npm i` without removing it